srcpkgs/kmscon/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'kmscon'
 pkgname=kmscon
-version=8
-revision=7
-build_style=gnu-configure
-configure_args="--disable-static --disable-multi-seat"
+version=9.0.0
+revision=1
+build_style=meson
+configure_args="-Dwerror=false"
 hostmakedepends="pkg-config docbook-xsl xkeyboard-config libxslt"
 makedepends="MesaLib-devel pango-devel libtsm-devel libxkbcommon-devel
  libpciaccess-devel eudev-libudev-devel dbus-devel fuse-devel"
 depends="xkeyboard-config"
 short_desc="Terminal emulator based on Kernel Mode Setting (KMS)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
-homepage="http://www.freedesktop.org/wiki/Software/kmscon"
-distfiles="${FREEDESKTOP_SITE}/kmscon/releases/kmscon-${version}.tar.xz"
-checksum=0ab01c1cdc4fbd692ce765e80478bc2d9663a7c55a5c75cc7ac421366ee6ae2b
+homepage="https://github.com/Aetf/kmscon"
+distfiles="https://github.com/Aetf/kmscon/archive/refs/tags/v${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/Aetf/kmscon/archive/v${version}.tar.gz"
```

Reply:
Please use refs/tags, just archive/ can return an error when there is a branch of the same name as tag